cloudslang-entities/src/main/java/io/cloudslang/lang/entities/ForLoopStatement.java

```diff
@@ -15,6 +15,8 @@
 
 public abstract class ForLoopStatement implements Serializable{
 
+    private static final long serialVersionUID = -1467375536631244911L;
+
```

Comment:
I thought it is best practice to generate serialVersionUID.. and I saw it is generated in other classes also (e.g. Input or InOutParam).. Should I remove them?